typescript/sdk/src/ism/utils.ts

```diff
@@ -170,11 +211,19 @@ export async function moduleMatchesConfig(
   multiProvider: MultiProvider,
   contracts: HyperlaneContracts<ProxyFactoryFactories>,
   mailbox?: Address,
+  // Less stringent checks on Routing/Pausable ISMs if not configured yet
+  configured = true,
```

Comment:
I don't think we should be modifying this function, nor using it in your code

Reply:
nice catch, leftover from previous iterations - I'm not using this fn anymore